dcicutils/qa_utils.py

```diff
@@ -829,7 +829,7 @@ def get_credentials(self) -> Optional[Boto3Credentials]:
         aws_access_key_id, aws_secret_access_key, _ = self._read_aws_credentials_from_file(aws_credentials_file)
         if aws_access_key_id and aws_secret_access_key:
             return Boto3Credentials(access_key=aws_access_key_id, secret_key=aws_secret_access_key)
-        return Boto3Credentials(access_key=None, secret_key=None)
+        return Boto3Credentials(access_key="", secret_key="")
```

Comment:
In my branch, I had done:
```
    MISSING_ACCESS_KEY = 'missing access key'
    MISSING_SECRET_KEY = 'missing secret key'
```
before the `def get_credentials` so that I could do (untested):
```suggestion
        return Boto3Credentials(access_key=self.MISSING_ACCESS_KEY, secret_key=self.MISSING_SECRET_KEY)
```

I think it's useful to easily recognize in the mocks when an important parameter isn't passed.